src/Integration.UnitTests/Suppression/SuppressedIssueMatcherTests.cs

```diff
@@ -111,10 +111,23 @@ public void MatchExists_MultipleServerIssues(string localRuleId, int localIssueL
             // Arrange
             var issueToMatch = CreateIssueToMatch(localRuleId, localIssueLine, localHash);
 
-            ConfigureServerIssues(issueToMatch,
-                CreateServerIssue("aaa", 222, "aaa hash"),
-                CreateServerIssue("bbb", 333, "bbb hash"),
-                CreateServerIssue("ccc", 444, "ccc hash"));
+            ConfigureServerIssues(
+                CreateServerIssue("aaa", 222, "aaa hash", isSuppressed: true),
+                CreateServerIssue("bbb", 333, "bbb hash", isSuppressed: true),
+                CreateServerIssue("ccc", 444, "ccc hash", isSuppressed: true));
+
+            // Act and assert
+            testSubject.SuppressionExists(issueToMatch).Should().Be(expectedResult);
+        }
+
+        [DataTestMethod]
+        [DataRow("RightRuleId", null, null, true, true)]
+        [DataRow("RightRuleId", null, null, false, false)]
```

Comment:
Minor: "Right" is potentially ambiguous - "Correct" or "Matching" are less ambiguous.